carvis-web/src/server/utils/lyft-helper.js

```diff
@@ -84,8 +81,27 @@ var lyftPhoneCodeAuth = function (fourDigitCode, phoneNumber, session, userLocat
       console.log('successful phoneCodeAuth post LYFT', data);
 
       // the responseMethod function returns an object with the parameters we need for subsequent operations only, and in a key-name generalised manner.
-      // TODO: DB POST // responseObject.dbUserProps (email, name etc.)
-      var response = lyftMethods.phoneCodeAuth.responseMethod(data);
+      var response = lyftMethods.phoneCodeAuth.responseMethod(data, userId);
+      var dbpostURL = 'https://54.183.205.82/users/update/:' + userId;
```

Comment:
are you supposed to put a colon in the url? i thought the colon was syntax in the route handler for expecting a path variable?